src/components/Navbar.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 import cx from 'classnames';
 import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import Checkbox from 'material-ui/Checkbox';
+
+
 import styles from '../styles/Navbar.css';
 
+
 export default function Navbar() {
+
```

Comment:
Would recommend you to install the eslint plugin to your editor, that way you will get feedback as you type into the editor. :)